lib/sequent/core/helpers/date_time_validator.rb

```diff
@@ -0,0 +1,22 @@
+require 'active_model'
+
+module Sequent
+  module Core
+    module Helpers
+      # Validates DateTimes
+      # Automatically included when using a
+      #
+      #   attrs value: DateTime
+      class DateTimeValidator < ActiveModel::EachValidator
+        def validate_each(subject, attribute, value)
+          return if value.is_a?(DateTime)
+          begin
+            DateTime.deserialize_from_json(value)
+          rescue
```

Comment:
I'm not 100% sure why we use rescue here. Some of our `deserialize_from_json` values do throw raises, and some do not. It depends on the implementation of the underlying objects. Integer, for example, tends to return 0 if it finds something weird:

`"".to_i
=> 0
`

It seems better to catch these type of things in `deserialize_from_json` and there to throw a better error, which we can then catch in this method.

My second comment is that `rescue` (implicitly all exceptions) is always a code smell for me.